### PR TITLE
Run tests binary with current_version=0.1

### PR DESCRIPTION
**Pull request by alexmalyshev**
Making sure nothing gets broken.  Some of the SetupResumeAcceptor tests were
failing as they very much assumed things to be 1.0.

### CODE REVIEW

File: test/internal/SetupResumeAcceptorTest.cpp [hunk 1]
```diff
@@ -138,56 +135,9 @@ TEST(SetupResumeAcceptor, SingleSetup) {
   EXPECT_TRUE(setupCalled);
 }
 
-TEST(SetupResumeAcceptor, SetupAndFnf) {
```
Comment (phoad) [inline test/internal/SetupResumeAcceptorTest.cpp]: Why did you delete this one?

Comment (alexmalyshev) [reply test/internal/SetupResumeAcceptorTest.cpp]: This test is invalid, actually.  It sends a SETUP and a REQUEST_FNF frame to a `OneFrameProcessor`, which isn't supposed to happen.
